install.sh

```diff
@@ -16,38 +16,116 @@
 
 # This is a simple installer script for KubeOne.
 
-set -eu
+set -e
 
-# What OS is used
-OS="$(uname | tr '[:upper:]' '[:lower:]')"
-# Find out what's the latest version
-VERSION="$(curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kubermatic/kubeone/releases | sed -n -E 's/^[[:space:]]*"tag_name": "v(([0-9])*\.([0-9])*\.([0-9])*)",$/\1/p' | sort -V | tail -1)"
-# Download URL for the latest version
-URL="https://github.com/kubermatic/kubeone/releases/download/v${VERSION}/kubeone_${VERSION}_${OS}_amd64.zip"
+KUBEONE_DIR="kubeone-tmp"
```

Comment:
```suggestion
KUBEONE_DIR=$(mktemp -d)
```

that will handle temp dir automatically